src/ngraph_builder.cc

```diff
@@ -374,6 +374,98 @@ static tf::Status TranslateAvgPoolOp(const tf::Node* op,
   return tf::Status::OK();
 }
 
+static tf::Status TranslateAvgPoolGradOp(const tf::Node* op, 
+                                     Builder::OpMap& ng_op_map) { 
+  TF_RETURN_IF_ERROR(ValidateInputCount(op, 2));
+      
+  shared_ptr<ng::Node> ng_grad;
+  TF_RETURN_IF_ERROR(GetInputNode(ng_op_map, op, 1, &ng_grad));
```

Comment:
Instead of this + `ValidateInputCount`, if you merge in latest `master` you can just do

`TF_RETURN_IF_ERROR(GetInputNodes(ng_op_map, op, nullptr, &ng_grad));`

Reply:
changed